taipy/core/data/in_memory.py

```diff
@@ -98,3 +98,16 @@ def _read(self):
 
     def _write(self, data):
         in_memory_storage[self.id] = data
+
+    def _append(self, data):
+        """Append data to the existing data in the in-memory storage."""
+        if self.id not in in_memory_storage:
+            in_memory_storage[self.id] = []
+
+        in_memory_storage[self.id].append(data)  # Append new data
```

Comment:
What if the previous data of the data node has no `append` method. 
In other words, what if `in_memory_storage[self.id]` has no `append` method?

Reply:
Ok I added a if instance condition to resolve that. Please review it and let me know if any changes required.